bin/i18n/i18n_script_utils.rb

```diff
@@ -68,7 +68,7 @@
 
 class I18nScriptUtils
   PROGRESS_BAR_FORMAT = '%t: |%B| %p% %a'.freeze
-  PARALLEL_PROCESSES = Parallel.processor_count / 2
+  PARALLEL_PROCESSES = Parallel.processor_count.freeze
```

Comment:
is there a reason for moving up the amount of parallel processes? I believe limiting the number of parallel processes was to avoid making too many requests to the crowdin API. And we are already running into request limit errors.

Reply:
We increased parallel processes to the most optimal amount for `sync-in` and `sync-out`.
`sync-up` uses custom configs based on [Crowdin API V2 Rate Limit](https://developer.crowdin.com/api/v2/#section/Introduction/Rate-Limits):
https://github.com/code-dot-org/code-dot-org/blob/65029dd3d441b6a34066f9aa62311b7155a8fd3b/bin/i18n/utils/sync_up_base.rb#L47